src/gas-hub.js

```diff
@@ -329,14 +382,15 @@ function showDiff(code, type) {
     })
     .concat(gasFiles)
     .filter(file => {
-      if (context.config.manifestEnabled && file === 'appsscript.json') {
+      const config = getConfig();
+      if (config.manifestEnabled && file === 'appsscript.json') {
         return true;
       }
-      for (let i = 0; i < context.config.ignorePattern.length; i++) {
-        let p = new RegExp(context.config.ignorePattern[i]);
-        if (p.test(file)) return false;
+      for (let i = 0; i < config.ignorePattern.length; i++) {
+        let p = new RegExp(svconfig.ignorePattern[i]);
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/083dcd77ddb6869d2b7dc2cdf2f7452c410d4901/src/gas-hub.js#L390"></span>‘p’ is assigned a value but never used.</td>
    </tr>
  </tbody>
</table>